mon-pix/app/controllers/authentication/login-or-register-oidc.js

```diff
@@ -35,6 +46,10 @@ export default class LoginOrRegisterOidcController extends Controller {
     return this.intl.primaryLocale;
   }
 
+  get authenticationKey() {
+    return this.oidcUserAuthenticationStorage?.authenticationKey ?? null;
```

Comment:
L'operateur [`.?`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining renvoie déjà un _Falsy_, pas besoin de chercher à renvoyer `null` :
) 

```suggestion
    return this.oidcUserAuthenticationStorage?.authenticationKey;
```